File: 80.lv/integrations/Java/OkHttp.java
OkHttpClient client = new OkHttpClient();

Request request = new Request.Builder()
	.url("https://api_dev.apitube.io/v1/sets/30054214?limit=250&offset=0")
	.get()
	.addHeader("X-ApiTube-Key", "YOUR-KEY-HERE")
	.build();

Response response = client.newCall(request).execute();

File: 80.lv/_integrations/Kotlin/OkHttp.kt
val client = OkHttpClient()

val request = Request.Builder()
	.url("https://apitube.io/v1/sets/55587074?limit=250&offset=0")
	.get()
	.addHeader("X-ApiTube-Key", "YOUR-KEY-HERE")
	.build()

val response = client.newCall(request).execute()
